portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +34,75 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> comments;
```

Comment:
Is there a reason for this to be a class variable? Looks like it is essentially only used in doGet (and used like a "global" variable to pass a parameter to writeGetResponse()).

Reply:
That's something I've been meaning to change. I used to store comments in that variable between calls to doGet to reduce my interactions with Datastore but it seemed like a bad practice because the servlet could get out of sync.